### ISSUE TITLE
Default time

### ISSUE THREAD

**Issue by khalilou88** [h2qutc/angular-material-components#6]
Hello,
Thanks for this lib.
I try to set a default time when selecting a datetime.
My use case is to set time at 00:00:00 when the user click on the button to select datetime. The user can change this default value if he want.

Thanks a lot

**Comment by h2qutc** [comment]
Hi, 
This repo is no longer supported, please consider using the repository [angular-material-components](https://github.com/h2qutc/angular-material-components) instead.
This issue was fixed in [@angular-material-components/datetime-picker](@angular-material-components/datetime-picker) v2.0.1

```
npm install --save  @angular-material-components/datetime-picker
```
Free free to open issue [here](https://github.com/h2qutc/angular-material-components/issues).
Thank you very much.

**Comment by kutensky** [comment]
You say that the issue was fixed. But I don't see a way to set default date and time

**Comment by h2qutc** [comment]
Hi, 
This issue is now fixed in the version v2.0.2. Feel free to check it out 👍 
The default times is passed by using Input defaultTime that is an array [hour, minute, second].
